src/plantPage/PlantCard.js

```diff
@@ -48,6 +63,14 @@ export function PlantCard({
           selection={selection}
           matchInfo={matchInfo}
         />
+
+        <Notes
+          value={plant.userNote}
+          onUserInput={onUserInput}
+          editMode={editMode}
+          plant={plant}
+          changeEditMode={handleEditMode}
```

Comment:
Here you can just write the function in the onClick directly, because it is so small. `<Notes
          value={plant.userNote}
          onUserInput={onUserInput}
          editMode={editMode}
          plant={plant}
          changeEditMode={() => setEditMode(!editMode)}/>`